base/abstractarraymath.jl

```diff
@@ -449,9 +449,10 @@ See also [`eachrow`](@ref), [`eachcol`](@ref), and [`selectdim`](@ref).
      This function requires at least Julia 1.1.
 """
 @inline function eachslice(A::AbstractArray; dims)
-    length(dims) == 1 || throw(ArgumentError("only single dimensions are supported"))
+    length(dims) == 1 || throw(ArgumentError("only single dimensions are supported in eachslice"))
     dim = first(dims)
```

Comment:
Could just change this to `only(dims)` and delete the check above.
That would change the error message to:
```
ERROR: ArgumentError: Collection has multiple elements, must contain exactly 1 element
Stacktrace:
 [1] only(::Array{Int64,1}) at ./iterators.jl:1117
```